src/posthog-core.js

```diff
@@ -83,6 +83,7 @@ const defaultConfig = () => ({
     inapp_link_new_window: false,
     request_batching: true,
     properties_string_max_length: 65535,
+    mask_all_inputs: false,
```

Comment:
As-is this option is confusing. We should clearly indicate this is session recording specific.

How about:

```
session_recording?: {
  mask_all_inputs: false
},
```